LEAF_Nexus/templates/view_homepage.tpl

```diff
@@ -256,6 +256,7 @@ $(function() {
             if(response != '') {
                 $('#currentEmployeeBody').html(response);
                 $('#currentEmployeeBody img').css('max-width', '64px');
+                $('#data_28_1_1').css('word-break', 'break-word');
```

Comment:
Either word-break: normal or overflow-wrap: anywhere would be better. break-word is deprecated.